.github/workflows/create_additional_release_tag.yaml

```
@@ -36,3 +36,10 @@ jobs:
           git tag $TAG_NAME
           git push origin $TAG_NAME
         done
+      # Generate a tag for unmanaged dependencies check.
+      # Use fixed tag so that checks in handwritten libraries do not need to
+      # update the version.
+        FIXED_CHECK_TAG="unmanaged-dependencies-check-latest"
```

```suggestion
        CHECK_LATEST_TAG="unmanaged-dependencies-check-latest"
```

The tag is not fixed. It's updated every time we make a release of sdk-platform-java.

---

done